app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1022,6 +1022,7 @@ def rbac_user_get_group_ids
   # Validate some of the user fields
   def rbac_user_validate?
     valid = true
+    Rails.logger.info("XXX: #{@edit[:new][:password]}, #{@edit[:new][:verify]}")
```

Comment:
Can this be removed or should it cleaned up to be helpful for debugging?

Reply:
Oh thanks... Was just testing stuff